assets/wizards/engagement/views/reader-activation/complete.js

```diff
@@ -107,12 +107,12 @@ export default withWizardScreen( () => {
 				setProgress( _progress => _progress + 1 );
 			}, generateRandomNumber( 1000, 2000 ) );
 		}
-		if ( progress === activationSteps.length && completed ) {
+		if ( progress >= activationSteps.length && completed ) {
 			setProgress( activationSteps.length + 1 ); // Plus one to account for the "Done!" step.
 			setProgressLabel( __( 'Done!', 'newspack-plugin' ) );
 			setTimeout( () => {
 				setInFlight( false );
-				// window.location = reader_activation_url;
+				window.location = reader_activation_url;
```

Comment:
I would recommend using `replace` here so we remove the 'setup complete' page from history.

```suggestion
				window.location.replace( reader_activation_url );
```

Feel free to ignore the suggestion above, but if you do, we should change this to `window.location.href`:

```suggestion
				window.location.href = reader_activation_url;
```

Reply:
Good idea! Added in 218eff70eb5f9a70a85ddb650ebe319cb6dbdb02